packages/lib/src/vis/shared/TooltipMesh.tsx

```diff
@@ -38,15 +38,13 @@ function TooltipMesh(props: Props) {
   // When panning, events are handled and stopped by texture mesh and do not reach this mesh (which is behind)
   const onPointerMove = useCallback(
     (evt: ThreeEvent<PointerEvent>) => {
-      const { zoom } = camera;
-      const projectedPoint = camera.worldToLocal(evt.unprojectedPoint.clone());
-
       const abscissaCoord = abscissaScale.invert(evt.unprojectedPoint.x);
       const ordinateCoord = ordinateScale.invert(evt.unprojectedPoint.y);
 
+      const projectedPoint = evt.unprojectedPoint.clone().project(camera);
```

Comment:
Previously, the point was projected from the world space into the local space.

I go one step further with `project` to project from the world space into the local space into the normalized space.